README.rst

```diff
@@ -2,21 +2,17 @@
 Lutris
 ******
 
-Lutris is an open source gaming platform for GNU/Linux.
-It makes gaming on Linux easier by managing, installing and providing optimal
-settings for games.
+Lutris is an open source gaming platform that makes gaming on Linux easier by
+managing, installing and providing optimal settings for games.
 
-Lutris does not sell games; you have to provide your own copy of the games
-unless they are open source or freeware.
-Games can be installed anywhere on your system; Lutris does not impose
-anything.
+Lutris does not sell games, and you must provide your own copy of a game
+unless it is open source/freeware.
+Instead, Lutris utilizes scripts called "runners" to provide a large library of
```

Comment:
nope, scripts are just scripts, runners are the programs that run the games (wine, retroarch, mame, ...) and the code associated with them.